### ISSUE TITLE
Blank warcraft recorder

### ISSUE THREAD

**Issue by Dumbmelee** [aza547/wow-recorder#234]
Warcraft recorder is just a blue square on each of the 3.X release versions (tried them all) for me.

It appears normal right after updating, but as soon as I click on any of the tabs like "Raids" it just goes blank and stays like that through relaunching it and all. It works on older release versions, just not on any of the 3.0 and beyond versions.

![image](https://user-images.githubusercontent.com/119078141/204051402-74e5b8da-a675-472e-ba8c-f2c6188b72a2.png)

**Comment by aza547** [comment]
See this comment:

![Screenshot_20221126-224008.jpg](https://user-images.githubusercontent.com/32389325/204111443-14037fb1-b3a2-4888-8023-9dc87ecb471a.jpg)

**Comment by aza547** [comment]
Tldr clear out your videos folder. A bit of a rubbish user experience - but it's expected over this upgrade - sorry! After you clear the folder, it shouldn't happen again. Please reopen if it doesn't resolve your problem.

**Comment by Dumbmelee** [comment]
> Tldr clear out your videos folder. A bit of a rubbish user experience - but it's expected over this upgrade - sorry! After you clear the folder, it shouldn't happen again. Please reopen if it doesn't resolve your problem.

Ah I could've just checked discord, thank you! :)
